docs/operation/upgrade.rst

```diff
@@ -9,11 +9,12 @@ Supported Versions
 ******************
 .. note::
 
-    MetalK8 supports upgrade **strictly** from one supported
-    minor version to another. For example:
+    MetalK8s supports upgrade of **at most** one minor version at a time.
+    For example:
 
-    - Upgrade from 2.0.x to 2.0.x
-    - Upgrade from 2.0.x to 2.1.x
+    - ✓ Upgrade from 2.4.0 to 2.4.4
+    - ✓ Upgrade from 2.4.0 to 2.5.1
+    - ✗ Upgrade from 2.4.0 to 2.6.0
```

Comment:
You have the exact same section in downgrade but maybe in another PR since we do not really care about downgrade for 2.6.0 (as you want)

Reply:
Makes me think... In `docs/conf.py`, we could load `salt/metalk8s/defaults.yaml`, and read the `metalk8s.downgrade.enabled` value to define a special Sphinx tag. This way, we could show a warning in the downgrade procedure for appropriate versions.
Would likely warrant another PR (unless you feel like flakies are dissuasive :stuck_out_tongue_winking_eye:)

Reply:
Done for downgrade section.
@gdemonet Great idea, did it here 53b881c7ca095f0c86d8e1090907018ebb5e1efe